src/test/unit/components/GameCard.test.tsx

```diff
@@ -2,34 +2,54 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { GameCard } from '../../../components/GameCard';
 
-// const img = "path-to-image"
-
-describe('Rendering GameCard', () => {
-  beforeEach(() => {
-    render(
-    <GameCard 
-        gameTitle='Dragon Age' 
-        gameDescription='Surprisingly few dragons.'
-        gameImage=''
-     />);
-  });
+describe('Rendering Game GameCard', () => {
+  
 
   it('renders GameCard', () => {
+    render(
+      <GameCard 
+          gameTitle='Dragon Age'
+          gameDescription='Surprisingly few dragons.'
+          gameImage={["//images.igdb.com/igdb/image/upload/t_thumb/jrumdkrrox09wbtblrka.jpg"]} 
+          gamePlatforms={["PS3"]} 
```

Comment:
Is it actually on the PS3 I had no idea

Reply:
me and my brother own it on ps3! never finished it though